### DISCUSSION TITLE
The Artillery. Mobile Artillery, Mobile Missile Launchers, All Cheap long range units aren't suppose to be that cheap. and here's why.

### DISCUSSION THREAD

**Discussion by AmericanSniper** [Team-QUIET/QUIET-Community-Edition#217] [Polls]
I’m going to present a few arguments, followed by possible counterarguments, and then I’ll attempt to address them. In the end, you can either agree or disagree with me. Let’s begin.

I’ve played the game many times against the AI M28, and I’ve noticed that when the AI wins, it’s often due to artillery spam. To be honest, I didn’t like this—it feels unfair and unnatural. So, let’s discuss what artillery should represent in this game. Artillery is definitely an anti-structure unit, designed to make players with heavily fortified defenses suffer because they can’t move or respond quickly. But what do we actually see in the game? An artillery-shield creep strategy. Players build artillery with shields and a few defensive units, and that’s enough. When the enemy tries to engage, they simply pull back the artillery, and because artillery is cheap, they can spam a few defensive units to protect it.

1st Argument:
A player who builds artillery against an enemy without bunkers should be punished. Why and how? I propose making artillery more expensive, so that building it without a valid reason (like targeting static defenses) becomes a much riskier choice.

2nd Argument:
Artillery should have a proper countermeasure—something that can actually kill or outplay it. Economically, this means there should be units that can outperform artillery spam, making artillery lose in terms of efficiency.

3rd Argument:
Artillery doesn’t need to be present in every game. On wide, open maps, artillery should be a very rare option.

4th Argument:
FAF, after more than a decade of existence, has also come to this conclusion. Although this is technically an appeal to authority, I believe it’s still a valid point.

In summary, I want artillery to be less common in regular gameplay but more impactful when it’s actually needed. Implementing this could be challenging, but I suggest making T3 artillery the slowest unit in the T3 class, with a significant cost increase (around 1,200 mass and corresponding energy cost). I’d be fine with increasing their HP as well, just to balance the nerf (allowing them to survive a few shots, making artillery duels less one-sided). Additionally, I suggest lowering the fire rate but increasing the alpha damage by about 15-20%. This would make artillery gameplay more manageable, as it’s difficult for players to keep track of hundreds of projectiles during a game.

Here are some possible counterarguments to these changes:

1st Counterargument: "You’re just nerfing the AI by making a unit that the AI plays well worse."
No, this isn’t true. The AI doesn’t get nerfed just because the game changes. The AI plays well with any unit in the game, except for a few odd ones. The AI doesn’t only outperform players with artillery; it has a choice mechanism, and when it repeatedly opts for the same strategy, it’s a sign that the strategy is abusive. That’s it. The AI is simply showing us an exploitative strategy, but nerfing that strategy won’t weaken the AI itself. M28 is much smarter than you think and will simply switch to other, more reliable units.

2nd Counterargument: "Just build shields."
This is one of the most frustrating responses you can hear. Shields are not a countermeasure to artillery because a countermeasure needs to be economically viable. In reality, shields are not a cost-effective solution.

3rd Counterargument: "How will players break bunkers with artillery if artillery becomes more expensive but static defenses don’t? Won’t everyone just start building bunkers?"
First of all, bunker building isn’t being nerfed because:
A) Point defenses aren’t as effective against combat units as you might think.
B) Focusing on static defenses is punished by their inability to move, and they are more expensive. This means you control less of the map. If you play on a chokepoint map where one Cerberus turret can defend the entire entrance to your base, that’s is just map fault.
C) Artillery doesn’t really care how strong point defenses are because it still destroys them for free. In fact, I’m even arguing to make artillery more deadly against point defenses by increasing its alpha damage.

4th Counterargument: "What about T2 artillery (static)? Isn’t it getting buffed by this?"
Well, in some ways, maybe yes. But there’s an interesting point here. Usually, when a stronger player is winning, they push the weaker player, forcing them to give up territory—usually the middle of the map, which is rich in resources. I know I’m getting into a lot of details here, but the point is, when the defending side gives up the middle and starts building bunkers, they’re also giving up a lot of resources. So, if you’ve reached the point where regular units can’t break through and you’re stuck with static defenses, the stronger player will have an economic advantage, which allows them to build artillery without issue.
And also, you can’t just build bunkers far away at the front line, because this stretches your connection to the base. So, either you’ll have to build more point defenses than necessary—which is economically inefficient—or you’ll end up leaving part of the front yard to the enemy, which again means giving up territory.

5th Counterargument: "I just don’t like it. It’s my opinion, and I don’t want your change in the game."
Sure, your opinion is your opinion, but that’s not an argument. Congratulations—you’re wrong.

By the way, if you read that as recent discussion. you can find some replays marked by me as artillery abused games.
in relevant discord channel.

**Comment by rajderks** [comment]
I agree that most games right now are the AI making TML/Artillery and it's getting kinda annoying, as there is no viable counter except experimentals.

I wouldn't know how to fix this though.

**Comment by AmericanSniper** [comment]
+Argument for MML cost nerf, the MML effectivity against TML are exponential, that why you can easily defend yourself against few units, but definitely cannot do anything against 20 (5k mass) units, at that point you can hardly doing anything but building shields & defenses, which a believe comes to early if we conclude the actual cost of that unit. the game itself are strongly snowballing.

**Comment by AmericanSniper** [comment]
Fucking unfair, I'm not the one who start debating carefully, but in respond get increased temperature that moves away neutral people who would agree on that just because they want to stay away from fucking drama that is rising due to no arguments from az side.
